tools/plugins/com.liferay.ide.project.ui/templates/java-templates-refrence.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This reference is for OSGi v5 style annotations.  We should add one for OSGi v6 style which jus tlooks like

```
@Reference
private ${FooLocal}Service = ${fooLocal}Service;
```